HelloWars/BotClient/BotProxy.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Mime;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Serialization;
+
+namespace BotClient
+{
+    public class BotProxy
+    {
+        // http://mccomputer/super-mc
+        // http://mccomputer/super-mc/bot-info
+        // http://mccomputer/super-mc/perform-next-move
+
+        public BotProxy(string url)
+        {
+
+        }
+
+        public string GetAvatarUrl()
+        {
+            return @"C:\Users\mariusz.iwanski\Documents\hello_wars\HelloWars\Arena\Assets\TempFoto.png";
```

Comment:
This is temporary situation. Will be replaced when BotProxy Class will be properly implemented. Right now I replace it by other 'nameless' code.